### ISSUE TITLE
Editor accessibility issues

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#6776]
- ~~[] Activate Font/Background ColorPicker. When the color picker appears, press the 'Escape' key. The focus does not move back to the tool.~~ - this is a common pattern for all tools in the Editor widget. Upon closing the tool dialog the focus always returns to the selected range in the Editor editable area.
- [x] There is no way to navigate to Font/Background ColorPicker Popups and select color: https://github.com/telerik/kendo-ui-core/issues/6654
- ~~[] Font/Background ColorPicker button labels are not exposed to the screen reader.~~ - Both Font and Background ColorPickers have `title` attribute set specifying the tool time. That title is announced by screen readers after the announcement of the current value in the tool.
- [] Could not associate a label to the Editor component: https://github.com/telerik/kendo-ui-core/issues/6317 - will not be implemented at this stage.
- [] [Provide feedback to SR when tools have successfully completed action](https://github.com/telerik/kendo-ui-core/issues/6779) - will not be implemented at this stage.
- [x] "Create Table" tool could not be activated via the keyboard: https://github.com/telerik/kendo-ui-core/issues/6655
- [x] "Table Wizard" button on the "Create table" popup is not keyboard accessible: https://github.com/telerik/kendo-ui-core/issues/6655
- ~~[] On 'Table Wizard' modal navigate to background color picked filed by using the keyboard tab key. Enter and activate the background color picker field. When the color picker appears, navigate to the "Clear Color" button. It does not have an accessible label.~~ - The `Clear Color` button on the FlatColorPicker has title set which specifies its accessible name. There is no explicit need to have a dedicated label element or aria-label element, as the title is announced by screen readers. Moreover, aXe DevTools does not report any violation in this scenario.
